io/exportForGit.m

```diff
@@ -103,11 +103,12 @@
     disp('COBRA version cannot be found')
 end
 %Retrieve libSBML version:
-fid = fopen('tempModelForLibSBMLversion.xml','w+');
-fclose(fid);
-evalc('[~,~,libSBMLver]=TranslateSBML(''tempModelForLibSBMLversion.xml'',0,0)');
-libSBMLver=libSBMLver.libSBML_version_string;
-delete('tempModelForLibSBMLversion.xml');
+try
+    libSBMLver=OutputSBML;
```

Comment:
This function requires at least a model as input; the following error occurs otherwise (also in version 5.17.0):
```
Error using OutputSBML
Must supply at least the model as an input argument
USAGE: OutputSBML(SBMLModel, (filename), (exclusiveFlag), (applyUserValidation), (fbcGeneProductOptions))
```

Reply:
@BenjaSanchez it doesn't (as long as an output is specified), see [release notes](http://sbml.org/Software/libSBML/LibSBML_Release_Notes):

- The libSBML-matlab interface function OutputSBML can now be called with an output argument and no inputs to return information about the libSBML version and enabled packages used to build the binary files.

Make sure that your libSBML is 5.17.0, with `which OutputSBML`.

Reply:
@edkerk you are correct, in my case `OutputSBML` is called from cobratoolbox, as they keep a binary of it as well. That's also why it still takes long to save for me. Hopefully soon they will update it, but in the meantime great that you included the amend :)